### PR TITLE
Fix event test

### PR DESCRIPTION
**Pull request by akry**

### CODE REVIEW

File: vnet/lib/vnet/models/tunnel.rb [hunk 1]
```diff
@@ -12,5 +12,11 @@ class Tunnel < Base
     many_to_one :src_interface, :class => Interface, :key => :src_interface_id
     many_to_one :dst_interface, :class => Interface, :key => :dst_interface_id
 
+    def self.create(options)
+      super(options) if self.find(src_datapath_id: options[:src_datapath_id],
```
Comment (hansode) [inline vnet/lib/vnet/models/tunnel.rb]: [reek] [NilCheck] performs a nil-check.

File: vnet/lib/vnet/models/tunnel.rb [hunk 2]
```diff
@@ -12,5 +12,11 @@ class Tunnel < Base
...
+                                  dst_datapath_id: options[:dst_datapath_id],
+                                  src_interface_id: options[:src_interface_id],
+                                  dst_interface_id: options[:dst_interface_id]).nil?
```
Comment (Metallion) [inline vnet/lib/vnet/models/tunnel.rb]: To be honest this feels a bit like a dirty fix. It shouldn't be the create method's responsibility to check if a tunnel exists already. Why is the create method being called for an existing tunnel?

**Comment by Mopster** [comment]
:+1:

**Comment by Metallion** [comment]
This should be merged into develop, not master.

**Pull request closed without merging**